testing/conftest.py

```diff
@@ -0,0 +1,3 @@
+def pytest_addoption(parser) -> None:
+    parser.addoption("--mqtt_host", action="store", default="localhost", help="mqtt host to be connected through")
+    parser.addoption("--mqtt_port", action="store", default=1883, help="mqtt port to be connected through")
```

Comment:
What about conveying `type=int`?
-- https://docs.python.org/3/library/argparse.html#type

```suggestion
    parser.addoption("--mqtt_port", action="store", type=int, default=1883, help="mqtt port to be connected through")
```